airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py

```diff
@@ -1254,6 +1259,22 @@ def _validate_stream_statuses(configured_catalog: ConfiguredAirbyteCatalog, stat
             assert status_list[-1] == AirbyteStreamStatus.COMPLETE
             assert all(x == AirbyteStreamStatus.RUNNING for x in status_list[1:-1])
 
+    @staticmethod
+    def _validate_state_messages(state_messages: List[AirbyteMessage]):
+        for state_message in state_messages:
```

Comment:
Let's add an additional assertion that there is at least one state message. As part of https://github.com/airbytehq/airbyte-internal-issues/issues/2925 all syncs that end in success should have emitted at least one state message

Reply:
Added an additional assertion